src/veccs/orderings.py

```diff
@@ -348,3 +349,59 @@ def maxmin_cpp_ancestor(
         ancestor_set_reduced=ancestorApprox,
     )
     return ordering
+
+def find_nns_l2_mf(locs_all: List[np.ndarray], max_nn: int = 10) -> np.ndarray:
```

Comment:
the type for a python list is `list[]`. `typing.List[]` is deprecated since python 3.9 (see [PEP585](https://peps.python.org/pep-0585/)).